scripts/tfw_lib.sh

```diff
@@ -36,27 +36,32 @@ calc()
 distribute_queues()
 {
 	dev=$1
-	RXQ_MAX=$(ethtool -l $dev 2>/dev/null \
-		  | grep -m 1 RX | sed -e 's/RX\:\s*//')
+	RXQ_MAX=$2
 
-	echo "...distribute $dev queues"
+	echo "...set rx channels to $RXQ_MAX, please wait..."
+	# Set maximum number of available channels for better
+	# packets hashing.
+	ethtool -L $dev rx $RXQ_MAX >/dev/null 2>&1
```

Comment:
I tried to do this with hands
```
root@9011:~# ethtool -L enp7s0 rx 128
rx unmodified, ignoring
no channel parameters changed, aborting
current values: tx 128 rx 8 other 0 combined 0
```
So we have the issue #1073 - we can not set the required number of channels and we shall not just silently ignore the error. Instead, please print a warning message so we'll be able to catch issues like #1073 quicker in future.

Reply:
By the way why does the command fails?

Reply:
Yes - agree, error message should be added.
Command fails because the count of queues had already been set to 128 before the command invocation - so, repeated attempt for the same value is rejected.

Reply:
Added error message.